sequencing/vcf_pca/Readme.md

```diff
@@ -1,21 +1,18 @@
-<!-- dx-header -->
 # Calculate PCs from VCF (DNAnexus Platform App)
 
-Uses PLINK 1.9 and eigenstrat 6.0 to calculate principal components from VCF or PLINK bed/bim/fam files.
+## What does this applet do?
 
-This is the source code for an app that runs on the DNAnexus Platform.
-For more information about how to run or modify it, see
-https://wiki.dnanexus.com/.
-<!-- /dx-header -->
+This app uses PLINK 1.9 and eigenstrat 6.0 to calculate principal components from VCF (`*.vcf.gz`) or PLINK files(`*.bed`, `*.bim`, `*.fam`). The app can process both type of data at the same time.
 
+## What data are required for this app to run?
 
+This app takes the following inputs:
 
-<!--
-TODO: This app directory was automatically generated by dx-app-wizard;
-please edit this Readme.md file to include essential documentation about
-your app that would be helpful to users. (Also see the
-Readme.developer.md.) Once you're done, you can remove these TODO
-comments.
+For VCF data, this app requires equal number of VCF (variant calling format) files(`*.vcf.gz`) and their associated index files (`*.vcf.gz.tbi`). VCF is the standard format for variant calling. 
```

Comment:
Change to "For VCF data, this app requires an equal number of VCF (variant calling format) files (`*.vcf.gz`) and their associated index files (`*.vcf.gz.tbi`). VCF is the standard format for variant calling."

Also, do we have to explain what a VCF file is?